content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+![AllPods](/images/guides/kots/ha-cluster-all-pods-one-node.png)
+
+If you are following along, you should be able to access the application by browsing to http://<ip-address-of-ha-proxy>. The sample app uses flask 'routes' to call various methods to interact with Postgres. To test if it can write to the database, use the  `/sql-check`   route which will check connection to the default database and list the databases are avaialble to the 'postgres' user. To write to the database, first use the `/sql-create` route which creates a database (appdiretdb) with a table (tblrecords) in it. Finally, use the `/sql-add` route to add a row the table with the timestamp. Once the database and table are created, simply use this route to add more records to the database.
+
+To verify that the data is in fact being written to Postgres, install pgAdmin and create a connection to the database using the Load Balancer's IP address. Once conneted, you should see the database and table. To retrieve records you can run a new query like this:
+
+`SELECT * FROM tblrecords`
+
+You should see records for each time that the /sql-add route is accessed. Below is a screenshot of what it should look like:
+
+![pgAdmin](/images/guides/kots/ha-cluster-pg-admin.png)
+
+### Adding Remaining Nodes to Cluster
+
+Under Cluster Management you can view the status of the embedded cluster. At this time, it only has one node. 
+
+
+![ClusterManagement](/images/guides/kots/ha-cluster-cluster-mgmt.png)
+
+To get the commands to run on the other nodes, click on the 'Add Node' button. From here you can retrieve the command to use on Worker and Master nodes. For our exercise we will add Master nodes to give us a proper HA solution.
+
+![ClusterManagement](/images/guides/kots/ha-cluster-add-node-cmd.png)
+
+Copy the command and then ssh into nodes 02 & 03 and paste the command and hit enter. This should run a process similar to the initial install of the cluster and will take a few minutes. Once it is finished you should see something similar to the output below:
+
+![ClusterManagement](/images/guides/kots/ha-cluster-node-joined.png)
+
+### Verify Pods Are Running on All Nodes
+
+To verify that the nodes have in fact joined the cluster you can run the following commands:
+
+```shell
+kubectl get nodes
+```
+The output should show us all three nodes of our cluster and have a status of `READY`.
+
+```shell
+kubectl get pods --all-namespaces -o wide
+```
+
+The output should show that other than the default namespace, there are pods running on all three nodes.
+
+# Part II: Testing Scenarions & Troubleshooting
+
+There are various testing scenarios for an HA Cluster. For starters we are focusing on data retention/loss scenario. The sample application is already configured to write to the embedded Postgres StatefulSet.
```

Comment:
I think this needs a tweak but I don't know what. Tried with `the` lemme know what you think 


```suggestion
There are various testing scenarios for an HA Cluster. For starters we are focusing on the data retention/loss scenario. The sample application is already configured to write to the embedded Postgres StatefulSet.
```